sktime/forecasting/compose/_reduce.py

```diff
@@ -895,15 +895,25 @@ def _predict_last_window(
             fh_max = fh.to_relative(self.cutoff)[-1]
 
             y_pred = np.zeros(fh_max)
+
+            # Array with input data for prediction.
             last = np.zeros((1, n_columns, window_length + fh_max))
 
             # Fill pre-allocated arrays with available data.
             last[:, 0, :window_length] = y_last
             if X is not None:
-                last[:, 1:, :window_length] = X_last.T
-                last[:, 1:, window_length:] = X.iloc[
-                    -(last.shape[2] - window_length) :, :
-                ].T
+                X_to_use = np.concatenate(
+                    [X_last.T, X.iloc[-(last.shape[2] - window_length) :, :].T], axis=1
+                )
+                if X_to_use.shape[1] < window_length + fh_max:
```

Comment:
there is no "equals" case, is that fine?

Reply:
I assume so, because then `X_to_use` needs no modification, right?

Reply:
Exactly. I will add a comment on it to avoid confusion. Or should I add even a dummy else statement?

Reply:
comments are ok, you can even write sth like, incl the comment lines
```python
# else X_to_use.shape[1] == window_length + fh_max
# and there are no additional steps to take
```

Reply:
Done